### PR TITLE
Update Interoperability module asset creation 

### LINKED ISSUE #84: Update Interoperability module asset creation
### Description

PR https://github.com/LiskHQ/lips/pull/274 modifies the procedure Interoperability module's asset creation. The corresponding changes must be reflected in the Migrator implementation.

### Acceptance Criteria

- The implementation for the following methods are updated:
  - `addInteropModuleEntry` (Ref schema available [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md#genesis-assets-schema))
- Unit tests are updated accordingly

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #84 #102  

### How was it solved?

- [x] Update Lisk SDK to v6.0.0-alpha.22
- [x] Update Interoperability module asset creation  
  - [x] Update `addInteropModuleEntry` method
  - [x] Update type and schema
  - [x] Update unit test

### How was it tested?
Local + unit test

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -312,59 +303,29 @@ export interface TerminatedOutboxAccount {
...
+export interface TerminatedStateAccounts {
+	chainID: Buffer;
+	terminatedStateAccount: TerminatedStateAccount;
+}
 
-export type RegisteredNamesSubstore = {
-	storeKey: Buffer;
-	storeValue: ChainID;
-}[];
+export interface TerminatedOutboxAccounts {
```
Comment (sameersubudhi) [inline src/types.ts:318]:
```suggestion
export interface TerminatedOutboxAccount {
```

File: src/types.ts [hunk 2]
```diff
@@ -312,59 +303,29 @@ export interface TerminatedOutboxAccount {
...
-	chainValidatorsSubstore: ChainValidatorsSubstore;
-	ownChainDataSubstore: OwnChainDataSubstore;
-	terminatedStateSubstore: TerminatedStateSubstore;
-	terminatedOutboxSubstore: TerminatedOutboxSubstore;
-	registeredNamesSubstore: RegisteredNamesSubstore;
+	ownChainName: string;
+	ownChainNonce: bigint;
+	chainInfos: ChainInfos[];
+	terminatedStateAccounts: TerminatedStateAccounts[];
+	terminatedOutboxAccounts: TerminatedOutboxAccounts[];
```
Comment (sameersubudhi) [inline src/types.ts:328]:
```suggestion
	chainInfos: ChainInfo[];
	terminatedStateAccounts: TerminatedStateAccount[];
	terminatedOutboxAccounts: TerminatedOutboxAccount[];
```

File: src/assets/interoperability.ts [hunk 3]
```diff
@@ -11,48 +11,18 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
-import {
-	MODULE_NAME_INTEROPERABILITY,
-	CHAIN_ID_LENGTH,
-	CHAIN_NAME_MAINCHAIN,
-	EMPTY_BYTES,
```
Comment (sameersubudhi) [inline src/assets/interoperability.ts:18]: Please check and remove any unnecessary constants from the file, if they're not being used elsewhere.

File: src/types.ts
Comment (sameersubudhi) [inline src/types.ts:1]: Please adjust the interface names wherever applicable.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

**Review by sergeyshemyakov — APPROVED** [review]
The Interoperability genesis logic is correct.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**